packages/components/src/composite/stories/index.story.tsx

```diff
@@ -60,27 +56,33 @@ const meta: Meta< typeof UseCompositeStorePlaceholder > = {
 };
 export default meta;
 
-export const Default: StoryFn< typeof Composite > = ( { ...initialState } ) => {
+export const Default: StoryFn< typeof Composite.Root > = ( {
+	...initialState
+} ) => {
 	const rtl = isRTL();
-	const store = useCompositeStore( { rtl, ...initialState } );
+	const store = Composite.useStore( { rtl, ...initialState } );
 
 	return (
-		<Composite role="grid" store={ store } aria-label="Ariakit Composite">
-			<CompositeRow role="row">
-				<CompositeItem role="gridcell">Item A1</CompositeItem>
-				<CompositeItem role="gridcell">Item A2</CompositeItem>
-				<CompositeItem role="gridcell">Item A3</CompositeItem>
-			</CompositeRow>
-			<CompositeRow role="row">
-				<CompositeItem role="gridcell">Item B1</CompositeItem>
-				<CompositeItem role="gridcell">Item B2</CompositeItem>
-				<CompositeItem role="gridcell">Item B3</CompositeItem>
-			</CompositeRow>
-			<CompositeRow role="row">
-				<CompositeItem role="gridcell">Item C1</CompositeItem>
-				<CompositeItem role="gridcell">Item C2</CompositeItem>
-				<CompositeItem role="gridcell">Item C3</CompositeItem>
-			</CompositeRow>
-		</Composite>
+		<Composite.Root
+			role="grid"
+			store={ store }
+			aria-label="Ariakit Composite"
```

Comment:
Why are we labeling this "Ariakit Composite"? I expected it to be just `Composite`:

```suggestion
			aria-label="Composite"
```

Reply:
Not sure either, it's a remnant from previous work.